packages/analytics-types/src/event.ts

```diff
@@ -107,11 +113,8 @@ export interface TrackEvent extends BaseEvent {
 
 export interface IdentifyEvent extends BaseEvent {
   event_type: SpecialEventType.IDENTIFY;
-  user_properties:
-    | IdentifyUserProperties
-    | {
-        [key in Exclude<string, IdentifyOperation>]: any;
-      };
+  global_user_properties?: UserProperties;
```

Comment:
Are we 100% final on this property being part of the event definition? Naming, etc? Changing it after it's in production will be hard.

If this feature is in beta only I would prefer to leaving the base types unchanged.

TS has "[declaration merging](https://github.com/amplitude/Amplitude-Kotlin/pull/144#discussion_r1323554003)" for interfaces. Maybe you could define another `interface IdenitfyEvent { global_user_properties?: UserProperties; } ` in the plugin package for now?
Or type cast the event to `any` in the plugin for assignment `(event as any).global_user_properties = user_properties`

Reply:
oh, I like this idea! I'll do that, especially since this would limit this definition to this plugin, and I'm not sure if we're driving this feature outside of our org.